src/main/resources/db/migration/V1_14__prisoner_iep_current_true_constraint.sql

```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX one_booking_id_with_current_true_idx ON prisoner_iep_level (booking_id) WHERE current = true;
```

Comment:
Nice. Would this work on the table which currently probably has some rows violating this? (don't know if it would fail to create the index/migrate or it would instead create this fine)

Reply:
That's a good point - I'll start up a local Postgres instance and try running the migration against it to see what happens

Reply:
hmm very good point. maybe we need a cleanup step above?

Reply:
Indeed - attempting to run that migration against a DB with existing rows violating this results in an error
```
Caused by: org.flywaydb.core.internal.sqlscript.FlywaySqlScriptException: Migration V1_15__prisoner_iep_current_true_constraint.sql failed
----------------------------------------------------------------
SQL State  : 23505
Error Code : 0
Message    : ERROR: could not create unique index "one_booking_id_with_current_true_idx"
  Detail: Key (booking_id)=(1234567) is duplicated.
```
Will